static/js/settings_user_groups.js

```diff
@@ -255,6 +208,60 @@ exports.populate_user_groups = function () {
     });
 };
 
+exports.set_raw_description = function (target, destination) {
+    var group_id = $(target).closest('.user-group').attr("id");
+    var group_data = user_groups.get_user_group_from_id(group_id);
+    if (!group_data) {
```

Comment:
I think we don't require this if condition here, cause in `user_groups.get_user_group_from_id(group_id)` if group_id do not exist, it will throw an error.

Reply:
removed it :+1: